app/infrastructure/messaging/websocket_manager.py

```
@@ -1 +1,41 @@
+            # Clean up empty lists to save memory
+            if not self.active_connections[user_id]:
+                del self.active_connections[user_id]
+
+    async def broadcast_to_user(self, user_id: str, message: dict):
+        # Sends a JSON message to all active devices for a specific user.
+        if user_id in self.active_connections:
+            # Iterate through all open tabs/apps for this user
+            for connection in self.active_connections[user_id]:
+                try:
+                    await connection.send_json(message)
+                except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)